hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestKeyDeletingService.java

```diff
@@ -137,6 +137,10 @@ public void checkIfDeleteServiceWithFailingSCM()
 
     final int keyCount = 100;
     createAndDeleteKeys(keyManager, keyCount, 1);
+    // Wait for the KeyDeletingService to pick up all the keys.
+    // There isn't any time-consuming code after the deleteKey call - which can
+    // cause assertion failures.
+    Thread.sleep(2000);
```

Comment:
Instead of adding fix sleep, it is better to check repeatedly with `GenericTestUtils.waitFor`, which lets the test pass sooner if the condition is met.  This allows increasing max. wait time without affecting usual run time.

Reply:
Thanks for the correction @adoroszlai, I have fixed this is in my latest patch